docs/src/components/productDesignKit/DesignKitFAQ.tsx

```diff
@@ -14,6 +13,16 @@ import InternalLink from 'docs/src/modules/components/Link';
 import Section from 'docs/src/layouts/Section';
 
 const faqData = [
+  {
+    summary: 'How long until MUI Connect is out?',
```

Comment:
it feels like we should call it Material UI Connect at this point. There is nothing beyond MUI yet.

Reply:
I'm not sure I'd agree — I think this would limit it to Material UI too much. I'd rather say it's a general plug-in that's just initially optimized for Material UI but with a horizon to work with other Figma design kits in the future (open to the possibility). Though, in hindsight, I'd definitely cut the "MUI" prefix :)

Reply:
> with a horizon to work with other Figma design kits in the future (open to the possibility). Though, in hindsight, I'd definitely cut the "MUI" prefix :)

Yes, same conclusion, if it's beyond MUI, feels strange to have MUI 👍 (I'm pushing the same to remove MUI from MUI Toolpad).

Reply:
Or we can start off skipping the rename chore and say it's initially optimized for the Material UI kit but will be available for others in the future! :)

Reply:
Why not

@DavidCnoops Do you have a preferred route?